### DISCUSSION TITLE
Welcome to athom-configs Discussions!

### DISCUSSION THREAD

**Discussion by athom-tech** [athom-tech/athom-configs#44] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by babakskr** [comment]
Anyone can help me with the homebridge user password.  Im trying to update nodes js but when I try is required password to @homebridge user. 

I tried my password, the default admin password. And nothing

Any one can help me? Thanks
